code/server/src/models/connection.ts

```diff
@@ -11,6 +14,34 @@ const config: object = {
   database: process.env.DB_DATABASE,
 };
 
-export const connection = mysql.createConnection(config);
+export class Connection {
+  private static __connection: mysql.Connection|null = null;
+
+  private static __createConnection = () => {
+    const connection = mysql.createConnection(config);
+    connection.on('error', err => {
+      if (err && err.fatal) {
```

Comment:
Create standalone error handler in connection.ts.

Reply:
Fixed [here](https://github.com/Fezzzi/Household-Chores/pull/10/commits/df91c2a3b76cb0477266a4db46e1397cbb108b9f#diff-f509ebf1c6486e48524aac270ff642b7)